### PR TITLE
check hex string

### PR DESCRIPTION
**Pull request by Agnewee**

### CODE REVIEW

**Comment by timofurrer** [comment]
Awesome, thanks :tada: 
Can you add a simple unit test though?

**Comment by Agnewee** [comment]
Sorry I forgot it before, it already be added now.

File: colorful/utils.py [hunk 1]
```diff
@@ -17,6 +18,17 @@ def hex_to_rgb(value):
...
-    return tuple(int(value[i:i + int(length / 3)], 16) for i in range(0, length, int(length / 3)))
+
+    check_hex(value, length)
+
+    step = int(length / 3)
+    return tuple(int(value[i:i+step], 16) for i in range(0, length, step))
+
+
+def check_hex(value, length):
+    if re.search(r'[g-z]', value, re.I):
```
Comment (timofurrer) [inline colorful/utils.py]: I'd check if all characters are valid instead of one invalid. 
In this implementation `......` would be treates as valid.

Comment (Agnewee) [inline colorful/utils.py]: But I think, It should raise Error whether one invalid or more

Comment (timofurrer) [inline colorful/utils.py]: I thought about something like that: 

```python
if not re.match(r'[0-9a-f]+', ...):
    raise ValueError(...)
```

(I am on vacation and online with my phone - so I cannot test or support better. Sorry) 

:beers:

Comment (Agnewee) [inline colorful/utils.py]: You're right. I'll fix it.

Comment (Agnewee) [inline colorful/utils.py]: You're right, I'll fix it.

**Comment by timofurrer** [comment]
Thanks :tada:

**Pull request merged**